src/dcap_provider.cpp

```diff
@@ -714,3 +721,112 @@ extern "C" sgx_plat_error_t sgx_ql_set_logging_function(
     logger_callback = logger;
     return SGX_PLAT_ERROR_OK;
 }
+
+extern "C" sgx_plat_error_t sgx_get_qe_identity_info(
+    sgx_qe_identity_info_t** pp_qe_identity_info)
+{
+    std::string intel_cert_base_url = INTEL_QE_IDENTITY_URL;

```

Comment:
There's no need to create this string. I'd just pass `QE_IDENTITY_URL` everywhere.

Reply:
addressed